libs/types-common/src/Util/Test.hs

```diff
@@ -0,0 +1,59 @@
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell   #-}
+
+module Util.Test where
+
+import Data.Monoid
+import Data.Proxy
+import Data.Tagged
+import Data.Typeable
+import Options.Applicative
+import Test.Tasty
+import Test.Tasty.Options
+
+newtype ServiceConfigFile = ServiceConfigFile String
+    deriving (Eq, Ord, Typeable)
+
+instance IsOption ServiceConfigFile where
+    defaultValue = ServiceConfigFile "/etc/wire/service/conf/service.yaml"
+    parseValue = fmap ServiceConfigFile . safeRead
+    optionName = return "service-config-file"
+    optionHelp = return "Service config file to read from"
+    optionCLParser =
+      fmap ServiceConfigFile $ strOption $
+        (  long (untag (optionName :: Tagged ServiceConfigFile String))
+        <> help (untag (optionHelp :: Tagged ServiceConfigFile String))
+        )
+
+newtype IntegrationConfigFile = IntegrationConfigFile String
+    deriving (Eq, Ord, Typeable)
+
+instance IsOption IntegrationConfigFile where
+    defaultValue = IntegrationConfigFile "/etc/wire/conf/integration.yaml"
+    parseValue = fmap IntegrationConfigFile . safeRead
+    optionName = return "integration-config-file"
+    optionHelp = return "Integration config file to read from"
+    optionCLParser =
+      fmap IntegrationConfigFile $ strOption $
+        (  long (untag (optionName :: Tagged IntegrationConfigFile String))
+        <> help (untag (optionHelp :: Tagged IntegrationConfigFile String))
+        )
```

Comment:
`./dist/cargohold-integration -h`
```
Available options:
  -h,--help                Show this help text
  -p,--pattern ARG         Select only tests that match pattern
  -t,--timeout ARG         Timeout for individual tests (suffixes: ms,s,m,h;   default: s)
  --service-config-file ARG
                          Service config file to read from
  --integration-config-file ARG
                           Integration config file to read from
  ....
```
Could you add a short-hand (`-c`, `-i`), and perhaps shorten values to `--service-config` and `--integration-config`?